src/darwin/Framework/CHIP/templates/MTRCallbackBridge_internal.zapt

```diff
@@ -0,0 +1,102 @@
+{{> header}}
+
+#import <Foundation/Foundation.h>
+#import "MTRCallbackBridgeBase_internal.h"
+
+#include <app/data-model/DecodableList.h>
+#include <app/util/im-client-callbacks.h>
+#include <app-common/zap-generated/cluster-objects.h>
+
+typedef void (*CommandSuccessCallback)(void *, const chip::app::DataModel::NullObjectType &);
+using CommandSuccessCallbackType = CommandSuccessCallback;
+typedef void (*DefaultSuccessCallbackType)(void *);
```

Comment:
This lost various CHIP prefixes and did not grow MTR ones.  Same for the `chip_cluster_responses` callbacks....

That's probably fine, given that we did not have that prefixing on the enums or other things here, but just making sure that was intentional.